test/performance/odp_l2fwd.c

```diff
@@ -105,6 +108,8 @@ typedef struct {
 	uint64_t vec_tmo_ns;    /* Vector formation timeout in ns */
 	uint32_t vec_size;      /* Vector size */
 	int verbose;		/* Verbose output */
+	int packet_len;		/* Maximum packet length supported */
+	uint32_t seg_len;	/* Pool segment length */
```

Comment:
Minor thing, but these should be of the same type, it looks funny if they're not.